vignettes/rmd/Rcpp-libraries.Rmd

```diff
@@ -154,15 +154,24 @@ it.
 
 Given a basic package with C++ support, we can now turn to integrating the external
 package.  This complexity of this step can, as alluded to earlier, vary from very easy to
-very complex.  Simple cases include just depending on library headers which can either
-be copied to the package, or be provided by another package such as \pkg{BH}
-\citep{CRAN:BH}. It may also be a dependency on a fairly standard library available on
-most if not all systems.  The graphics formats bmp, jpeg or png may be example; text
-formats like JSON or XML are another.  One difficulty, though, may be that _run-time_
-support does not always guarantee _compile-time_ support. In these cases, a `-dev` or
-`-devel` package may need to be installed.
-
-In the concrete case of Corels, we
+very complex.  Simple cases include just depending on library headers which can either be
+copied to the package, or be provided by another package such as \pkg{BH} \citep{CRAN:BH}
+or \pkg{AsioHeaders} \citep{CRAN:AsioHeaders} or many other examples.
+
+One aspect worth noting is that if you include a type in your function interface it will
+also be part of the generated \code{RcppExports.cpp}. In this case adding a file
+\code{PACKAGE\_types.h} (where \code{PACKAGE} is to be replaced with the name of your
+package) containing the required \code{\#include} statement for the type(s) will permit
+compilation; see the 'Rcpp Attributes' vignette for details \citep{CRAN:Rcpp:Attributes}.
+
+It may also be a dependency on a fairly standard library available on most if
+not all systems.  The graphics formats bmp, jpeg or png may be example; text
```

Comment:
"may be example**s**", or "may be **an** example"

Reply:
Nice catch. That half-sentence is likely five years old.  Will fix.